core/src/org/pentaho/di/core/database/PostgreSQLDatabaseMeta.java

```diff
@@ -150,6 +150,11 @@ public boolean needsToLockAllTables() {
     return false;
   }
 
+  @Override
+  public String getSQLListOfSequences() {
+    return "SELECT relname AS sequence_name FROM pg_statio_all_sequences";
```

Comment:
Seems to me that the query should include pg_catalog:

From:
... snip... FROM pg_statio_all_sequences

To:
... snip... FROM pg_catalog.pg_statio_all_sequences